README.md

```diff
@@ -55,6 +56,22 @@ When the game is installed, you may run it with:
 lincity-ng
 ```
 
+## Alternative Binary Releases
+
+As an alternative to building from source, there are several places to get
+pre-built binary packages:
+
+- Linux distribution repositories. This is likely the best way to install
+  LinCity-NG pre-built binaries for Linux users. However, LTS and stable
+  distributions, by their very nature, may not offer the latest version of
+  LinCity-NG.
+- [Flathub](https://flathub.org/apps/io.github.lincity_ng.lincity-ng). As of
+  version 2.14.1, LinCity-NG is available on Flathub.
```

Comment:
Probably a good idea to add a short outline fo pros and cons here like it's done for other options? I.e.: likely to be more recent than a distro package on stable distributions, but probably bigger than other options due to nature of flatpak.